packages/app/src/components/syncNominations/syncNominations.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import nominationsAPI from '../../utils/API/nominationsAPI';
+import styles from './styles.css';
+
+const SyncNominations = () => {
+  const [message, setMessage] = useState('')
+
+  const handleClick = () => {
+    getSyncNom()
+  }
+
+  function removeMessage() {
+    setTimeout(() => {
+      setMessage('')
+    }, 2000)
+  }
+
+  function getSyncNom() {
+    nominationsAPI.syncNominations()
+    .then((res) => {
+      if (res.status === 200) {
+        setMessage('Nominations successfully synced')
+        // do we need to distinguish whether there's new data coming in or not?
```

Comment:
good question, did you raise this to Product?

Reply:
will do!